app/src/main/kotlin/co/netguru/baby/monitor/client/feature/babynotification/NotifyBabyEventUseCase.kt

```diff
@@ -15,14 +17,30 @@ class NotifyBabyEventUseCase(
     private val babyEvents: PublishSubject<BabyEvent> = PublishSubject.create()
 
     private fun fetchClientsAndPostNotification(babyEvent: BabyEvent): Completable {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/7c069c26782df05dac8a097aa956e351ad95f920/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/babynotification/NotifyBabyEventUseCase.kt#L19"></span>The function fetchClientsAndPostNotification appears to be too complex.</td>
    </tr>
  </tbody>
</table>